dist/ansible/scripts/ovn-setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+  
+# Create a configmap of the ovn configuration information
+# This script runs on the ovn-master node on the cluster
+# This is run as part of installation after the cluster is
+# up and before the ovn daemonsets are created.
+
+health=$(oc get --raw /healthz/ready)
+if [ $health != 'ok' ]
+then
+  echo " cluster not ready"
+  exit 1
+fi
+
+svcacct=$(oc get serviceaccount | grep ovn)
+if [ $? -ne 0 ]
+then
+  oc create serviceaccount ovn
+  oc adm policy add-cluster-role-to-user cluster-admin -z ovn
+  oc adm policy add-scc-to-user anyuid -z ovn
+fi
```

Comment:
For all the serviceaccount stuff, I believe we need to do this dance.  We can simply add the ClusterRole and bindings in the .yaml file that defines the daemonset, and the token will then be exposed in the ovn pod as a "secret" which we can grab from the filesystem.  In fact that's what InClusterConfig() does for us, except that we also need to get the apiserver address into it somehow.

But in any case, we can't use 'oc' here but most 'oc' commands can just be replaced by kubectl.

Reply:
OK I will change oc to kubectl and in a later version, when we do the daemonsets add the above. At present I am focused on getting an image.